plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -342,7 +342,8 @@ protected static Optional<Integer> getInteger(ResultSet resultSet, String column
     {
         int value = resultSet.getInt(columnLabel);
         if (resultSet.wasNull()) {
-            return Optional.empty();
+            // When the size is not obtained, the default setting is 65536, instead of throwing an exception to cause the query terminal, AnalyticDB for MySQL gets the size as empty
```

Comment:
We shouldn't put MySQL specific logic to `BaseJdbcClient`.

Reply:
This is not just mysql, it's the problem I commented on. I have also tested that data sources developed based on other engines such as postgres will also run abnormally because varchar does not specify the length. I think this kind of exception can be a warning instead of interrupting the query. Some engines do not specify the length. There is a default value.

Reply:
First of all, this method is just a helper method to get an integer from the provided label. Defaulting to 65536 totally doesn't make sense. 

> I have also tested that data sources developed based on other engines such as postgres will also run abnormally because varchar does not specify the length.

Then, please add tests.